internal/controller/operatorconfigmap_controller.go

```diff
@@ -793,6 +793,9 @@ func (c *OperatorConfigMapReconciler) reconcileRecipeOperatorSubscription() erro
 
 func (c *OperatorConfigMapReconciler) reconcileODFSnapshotterSubscription() error {
 	odfSnapshotterSubscription, err := c.getSubscriptionByPackageName("odf-snapshot-controller")
+	if kerrors.IsNotFound(err) {
+		odfSnapshotterSubscription, err = c.getSubscriptionByPackageName("odf-external-snapshotter-operator")
```

Comment:
pls reverse the search, have the d/s naming first as prod clusters have higher lifetime than u/s or test clusters.